digautoprofiler/profile_lib.py

```diff
@@ -7,37 +7,45 @@
 
 
 def getColumns(dfName: pd.DataFrame):
-    print(dfName.dtypes.to_json(default_handler=str))
+    index_dtype = pd.Series([dfName.index.dtype], index=["INDEX"])
+    combined_series = pd.concat([index_dtype,dfName.dtypes])
+    print(combined_series.to_json(default_handler=str))
 
 def getShape(dfName: pd.DataFrame):
     print(dfName.shape)
 
 def getQuantMeta(dfName: pd.DataFrame, colName: str):
-    m = dfName[colName].describe()
+    colData = dfName[colName] if colName != "INDEX" else dfName.index.to_series()
```

Comment:
Could you not do these if else's inline, I think it is more legible if multiple lines

Reply:
Done